typescript-magic-link-example/README.md

```diff
@@ -43,16 +43,24 @@ Node.js version 10+
 ```typescript
 WORKOS_API_KEY = your_api_key_here
 WORKOS_CLIENT_ID = your_project_id_here
+PORT = 8000
```

Comment:
All the other typescript apps use port 3000. Is there a reason we want to use 8000 here instead of keeping it consistent?